LogcatCoreLib/src/main/java/info/hannes/logcat/LogfileFragment.kt

```diff
@@ -16,7 +16,7 @@ class LogfileFragment : LogBaseFragment(), Observer<Event<String>> {
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         val view = super.onCreateView(inflater, container, savedInstanceState)
-        FileLoggingTree.lastLogEntry.observe(viewLifecycleOwner, this)
+        fileLoggingTree()?.lastLogEntry?.observe(viewLifecycleOwner, this)
```

Comment:
But this is suspect. Do you really create a new instance ? With a lowercase filename ?

Reply:
`fileLoggingTree() = Timber.forest().filterIsInstance<FileLoggingTree>().firstOrNull()`  is an extension function, we are not creating a new instance here.